docs/operations/export_api.md

```diff
@@ -217,8 +217,89 @@ The export server certificate is valid for 7 days after which it is rotated by d
+
+`vmexport` uses two mandatory arguments:
+1. The vmexport **function** (create|delete|download).
+2. The VirtualMachineExport **name**.
+
+These three **functions** are:
+
+#### Create
+```sh
+# Creates a VMExport object according to the specified flag.
+
+# The flag should either be:
+
+# --pvc, to specify the name of the pvc to export.
+# --snapshot, to specify the name of the VM snapshot to export.
+# --vm, to specify the name of the Virtual Machine to export.
+
+$ virtctl vmexport create name [flags]
+```
+
+#### Delete
+```sh
+# Deletes the specified VMExport object.
+
+$ virtctl vmexport delete name
+```
+#### Download
+```sh
+# Downloads a volume from the defined VMExport object.
+
+# The main available flags are:
+
+# --output, mandatory flag to specify the output file.
+# --volume, optional flag to specify the name of the downloadable volume.
+# --vm|--snapshot|--pvc, if specified, are used to create the VMExport object assuming it doesn't exist. The name of the object to export has to be specified.
+
+$ virtctl vmexport download name [flags]
+```
+
+For more information about usage and examples:
+
+```
+$ virtctl vmexport --help
+
+Export a VM volume.
+
+Usage:
+  virtctl vmexport [flags]
+
+Examples:
+# Create a VirtualMachineExport to export a volume from a virtual machine:
+	virtctl vmexport create vm1-export --vm=vm1
+
+	# Create a VirtualMachineExport to export a volume from a virtual machine snapshot
+	virtctl vmexport create snap1-export --snapshot=snap1
+
+	# Create a VirtualMachineExport to export a volume from a PVC
+	virtctl vmexport create pvc1-export --pvc=pvc1
+
+	# Delete a VirtualMachineExport resource
+	virtctl vmexport delete snap1-export
+
+	# Download a volume from an already existing VirtualMachineExport (--volume is optional when only one volume is available)
+	virtctl vmexport download vm1-export --volume=volume1 --output=disk.img.gz
+
+	# Create a VirtualMachineExport and download the requested volume from it
+	virtctl vmexport download vm1-export --vm=vm1 --volume=volume1 --output=disk.img.gz
+
+Flags:
+  -h, --help              help for vmexport
+      --insecure          When used with the 'download' option, specifies that the http request should be insecure.
+      --keep-vme          When used with the 'download' option, specifies that the vmexport object should not be deleted after the download finishes.
+      --output string     Specifies the output path of the volume to be downloaded.
+      --pvc string        Sets PersistentVolumeClaim as vmexport kind and specifies the PVC name.
+      --snapshot string   Sets VirtualMachineSnapshot as vmexport kind and specifies the snapshot name.
+      --vm string         Sets VirtualMachine as vmexport kind and specifies the vm name.
+      --volume string     Specifies the volume to be downloaded.
+
+Use "virtctl options" for a list of global command-line options (applies to all commands).
+```
```

Comment:
Nit: pasting the whole help here feels unnecessary and prone to rotting... I'd expect people reading this to have access to virtctl and be able to just type the help command.
Just my opinion, I'm fine keeping this too.

Reply:
I agree, however I also like to have all the available information in the user guide (some flags are only present here). If it's not a problem I'd prefer to keep it.